tests/unit_tests/test_game.py

```diff
@@ -91,6 +91,12 @@ def test_add_game_connection_throws_if_not_lobby_state(game: Game, players, mock
 
     game.on_game_end.assert_any_call()
 
+async def test_game_marked_dirty_when_timed_out(game: Game):
```

Comment:
Yeah, I wondered why the testing routine didn't do the sleep, now I see why :) I'll take a look